xbmc/addons/PVRClient.h

```diff
@@ -847,6 +847,18 @@ namespace PVR
     void OnPowerSavingDeactivated();
 
     /*!
+     * @brief Get the priority of this client. Higher value means lower priority.
```

Comment:
Personally I would have "higher value means higher priority", IMO it's much more logical

Reply:
Good point. Will change this.